ghcide/ghcide.cabal

```diff
@@ -45,77 +43,76 @@ common warnings
-    , binary
-    , bytestring
-    , case-insensitive
-    , co-log-core
-    , containers
-    , cryptohash-sha1              >=0.11.100 && <0.12
-    , data-default
-    , deepseq
-    , dependent-map
-    , dependent-sum
-    , Diff                         ^>=0.4.0
-    , directory
-    , dlist
-    , enummapset
-    , exceptions
-    , extra                        >=1.7.14
-    , filepath
-    , fingertree
-    , focus                        >=1.0.3.2
-    , ghc                          >=9.2
-    , ghc-boot
-    , ghc-boot-th
-    , ghc-check                    >=0.5.0.8
-    , ghc-paths
-    , ghc-trace-events
-    , Glob
-    , haddock-library              >=1.8      && <1.12
-    , hashable
-    , hie-bios                     ^>=0.14.0
-    , hie-compat                   ^>=0.3.0.0
-    , hiedb                        ^>= 0.6.0.0
-    , hls-graph                    == 2.8.0.0
-    , hls-plugin-api               == 2.8.0.0
-    , implicit-hie                 >= 0.1.4.0 && < 0.1.5
-    , lens
-    , list-t
-    , lsp                          ^>=2.5.0.0
-    , lsp-types                    ^>=2.2.0.0
-    , mtl
-    , opentelemetry                >=0.6.1
-    , optparse-applicative
-    , parallel
-    , prettyprinter                >=1.7
-    , prettyprinter-ansi-terminal
-    , random
-    , regex-tdfa                   >=1.3.1.0
-    , safe-exceptions
-    , sorted-list
-    , sqlite-simple
-    , stm
-    , stm-containers
-    , syb
-    , text
-    , text-rope
-    , time
-    , transformers
-    , unliftio                     >=0.2.6
-    , unliftio-core
-    , unordered-containers         >=0.2.10.0
-    , vector
+    Diff ^>=0.4.0,
+    Glob,
+    aeson,
+    array,
+    async,
+    base >=4.16 && <5,
+    base16-bytestring >=0.1.1 && <1.1,
+    binary,
+    bytestring,
+    case-insensitive,
+    co-log-core,
+    containers,
+    cryptohash-sha1 >=0.11.100 && <0.12,
+    data-default,
+    deepseq,
+    dependent-map,
+    dependent-sum,
+    directory,
+    dlist,
+    enummapset,
+    exceptions,
+    extra >=1.7.14,
+    filepath,
+    fingertree,
+    focus >=1.0.3.2,
+    ghc >=9.2,
+    ghc-boot,
+    ghc-boot-th,
+    ghc-check >=0.5.0.8,
+    ghc-paths,
+    ghc-trace-events,
+    haddock-library >=1.8 && <1.12,
+    hashable,
+    hie-bios ^>=0.14.0,
+    hie-compat ^>=0.3.0.0,
+    hiedb ^>=0.6.0.0,
+    hls-graph ==2.8.0.0,
+    hls-plugin-api ==2.8.0.0,
+    implicit-hie >=0.1.4.0 && <0.1.5,
+    lens,
+    list-t,
+    lsp ^>=2.5.0.0,
+    lsp-types ^>=2.2.0.0,
+    mtl,
+    opentelemetry >=0.6.1,
+    optparse-applicative,
+    parallel,
+    prettyprinter >=1.7,
+    prettyprinter-ansi-terminal,
+    random,
+    regex-tdfa >=1.3.1.0,
+    safe-exceptions,
+    sorted-list,
+    sqlite-simple,
+    stm,
+    stm-containers,
+    syb,
+    text,
+    text-rope,
+    time,
+    transformers,
+    unliftio >=0.2.6,
+    unliftio-core,
+    unordered-containers >=0.2.10.0,
+    vector,
```

Comment:
A lot of noise here from switching to trailing commas. I see that @soulomoon had a question about it too. Gild isn't (and hopefully won't be) configurable. I think trailing commas are generally preferable, especially when a final trailing comma is allowed (like it is in Cabal files for recent enough versions). It's a bit unfortunate in cases like this, but overall I think trailing commas are better.

Reply:
I'm fan of this approach (non-configurability).
Similar approach is taken by elm-format, which became standard within elm community.
I like that most libraries you can get is formatted with it so there's very consistent feel to the whole ecosystem.
It eliminates a lot of pointless bickering on what format to choose.
One gets used to it pretty quickly.

Reply:
Thanx for the reasonning, I am convinced that it is good.